src/util/linkify.js

```diff
@@ -12,8 +12,9 @@ const createLink = options => href => {
 	const target = options.target || '';
 	const targetAttr = `target="${target}"`;
 	const relAttr = target === '_blank' ? 'rel="noopener noreferrer"' : '';
+	const link = (href.search('www.') === 0) ? `http://${href}` : href;
```

Comment:
yeah, I was thinking that checking for the absence of any supported protocol is pretty trivial

```js
const hasProtocolRE = /^(?:https?:|ws:|ftp:)?\/\//;
const link = hasProtocolRE.test(href) ? href : `http://${href}`
```